tests/release/copy-application.sh

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env bash
```

Comment:
please add a note where this script comes from - I assume from [here](https://github.com/hacbs-release/release-utils)?

Reply:
this scrip is downloaded by test and not copied